fakebook/src/app/models/post.ts

```diff
@@ -0,0 +1,11 @@
+import { User } from './user';
+import { Comment } from './comment';
+
+export interface Post {
```

Comment:
these interfaces are still not documented. you should make up your mind whether or not they are meant to reflect JSON coming over the wire from the backends or not. it looks like they are 

if so, then you need to think about the REST design here. will every downloaded post include the details of a user, or just a user ID? the details of _all_ the comments, or should that be in a separate request? (ideally, the URLs are actually our IDs for REST, not just numbers) it mostly depends on what the frontend needs to know when.

`/posts` isn't going to be a very useful endpoint because that should fetch _all_ posts, and what posts a user sees on his feed should be related to who he follows, and so it's a different resource per-user and so the user needs to be reflected in the URL.

these things should have been already designed out